build.gradle

```diff
@@ -244,6 +244,8 @@ dependencies {
         }
     }
 
+    implementation group: "bio.terra", name: "terra-policy-client", version: "0.2.50-SNAPSHOT"
+    implementation group: "bio.terra", name: "terra-policy-service", version: "0.2.9-SNAPSHOT"
```

Comment:
It's interesting that we need to add a reference to both the client and the service - Could we get away with just using the client?

Reply:
Oh you're totally right! Including the service was what caused the weird build errors.